f5_openstack_agent/lbaasv2/drivers/bigip/listener_service.py

```diff
@@ -56,7 +56,9 @@ def create_listener(self, service, bigips):
             tls['name'] = vip['name']
             tls['partition'] = vip['partition']
 
+        network_id = service['loadbalancer']['network_id']
         for bigip in bigips:
+            self.service_adapter.get_vlan(vip, bigip, network_id)
```

Comment:
Alright.  I'm ok with this for now provided the traffic test passes for both 1-armed and 2-armed deployments.  Did you already run both variations?